src/UFS2ARCO/ufsdataset.py

```diff
@@ -57,22 +58,22 @@ class UFSDataset:
 
     chunks_in = None
     chunks_out = None
-    coords = None
-    data_vars = None
-    zarr_name = None
+    coords = []
+    data_vars = []
+    zarr_name = ""
```

Comment:
I actually prefer this to be None since UFSDataset does nothing on its own, and the zarr_name attribute is defined for FV3Dataset (and eventually for the ocean and ice models). This will eventually force an error if someone tries to use UFSDataset on its own.

Reply:
But it will be a string, yes?